cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -57,7 +57,8 @@ import org.ossreviewtoolkit.utils.perf
 import org.ossreviewtoolkit.utils.safeMkdirs
 
 class AnalyzerCommand : CliktCommand(name = "analyze", help = "Determine dependencies of a software project.") {
-    private val allPackageManagersByName = PackageManager.ALL.associateBy { it.managerName.toUpperCase() }
+    private val allPackageManagersByName = PackageManager.ALL.associateBy { it.managerName }
+        .toSortedMap(String.CASE_INSENSITIVE_ORDER)
```

Comment:
Just to be sure: Does this make the map actually `case-insensitive`, i.e., are "abc" and "ABC" hashed to the same value?

Reply:
Yes. I agree the name "CASE_INSENSITIVE_*ORDER*" is misleading, but in fact the keys become case-insensitive.